### DISCUSSION TITLE
Filter for File Tags

### DISCUSSION THREAD

**Discussion by wodka** [directus/directus#16044] [Feature Requests]
Right now it seems it is only possible to filter if a file has or does not have tags.

![image](https://user-images.githubusercontent.com/385731/196491776-3d3bf9fc-60f7-4770-a8a0-932e8e56b7f7.png)

It would be nice to instead of adding access tokens (which are basically impossible to handle through deployments) to be able to add a tag like "public" to then be able to filter that in the permissions.

![image](https://user-images.githubusercontent.com/385731/196491935-b99fc1d3-61f4-4dd3-bd47-505b08c8e72a.png)

**Comment by rijkvanzanten** [comment]
Alternatively, you can also add a new field to directus_files called `public` (just a toggle), or configure the permissions based on the folder the file is in, so you can have a `public` folder 👍🏻

**Reply by wodka** [reply]
both partial yes - are updates to internal tables covered by snapshots? (I have not tried)

Is it actually a bug that it is not possible to filter on tags?

**Reply by rijkvanzanten** [reply]
> both partial yes - are updates to internal tables covered by snapshots?

Not sure, but should be!

> Is it actually a bug that it is not possible to filter on tags?

No, filtering in json values (arrays or otherwise) is a new upcoming feature (#15889)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
